docker-compose.yml

```diff
@@ -1,20 +1,22 @@
 version: '3'
 
 services:
-  todoapp:
-    container_name: todoapp
+  tutorial-node:
+    container_name: nodetodo
     restart: always
+    image: rookout/tutorial-nodejs
     build: .
     environment:
-      - ROOKOUT_AGENT_HOST=rookout-agent
+    - ROOKOUT_AGENT_HOST=rookout-agent
```

Comment:
@zeevmoney you need to update this. Did you test it? :(

Reply:
Don't worry, I did - it works.
YAML doesn't care about indentation on this case.